client/checkout/blocks/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -129,7 +198,7 @@ const PaymentProcessor = ( {
 						type: 'success',
 						meta: {
 							paymentMethodData: {
-								payment_method: paymentMethodId,
+								payment_method: upeMethods[ paymentMethodId ],
```

Comment:
I think this is more consistent with other gateways: essentially we send the gateway ID (e.g. `woocommerce_payments`) instead of the Stripe ID (e.g. `card`) in the `$_POST['payment_method']` field. This is how I believe this parameter is intended to be used and have added a few other accommodations in the plugin to account for this.